docs/Makefile

```
@@ -1,31 +1,74 @@
+# The report is written to compare/output/ and served on http://127.0.0.1:8883.
+.PHONY: compare
+compare: compare-build compare-run
 
-.PHONY: push
-push:
-        $(RUNTIME) push $(IMG)
+.PHONY: compare-build
+compare-build: compare-build-base compare-build-head
+
+.PHONY: compare-build-base
+compare-build-base:
```

checkmake (0.3.2)

[warning] Target body for "compare-build-base" exceeds allowed length of 5 lines (9).

(maxbodylength)